daemon/src/model/cfd.rs

```diff
@@ -211,14 +211,20 @@ pub enum SettlementKind {
     Outgoing,
 }
 
+#[derive(thiserror::Error, Debug, PartialEq)]
+pub enum CannotAutoRollover {
+    #[error("Is too recent to auto-rollover")]
+    TooRecent,
+    #[error("Cfd does not have a dlc")]
+    NoDlc,
+}
```

Comment:
We went back and forth with these "cases" (`TooRecent` is not even a real error case, because you just can't rollover but there is nothing erroneous about that...).

For me, returning specific error types greatly helps the understanding of the logic, because I immediately see what is the problem. Furthermore, the caller can decide to react differently. Maybe there is a better way of modelling this, that does not actually build on errors. I would not invest the time to change them know though.

Reply:
I'm keeping them around for the time being.